olive/passes/onnx/genai_model_exporter.py

```diff
@@ -79,8 +98,8 @@ def _run_for_config(
             model_name=str(model.model_path or model.hf_config.model_name),
             input_path="",  # empty string for now
             output_dir=str(output_model_filepath.parent),
-            precision=str(precision),
-            execution_provider=str(device),
+            precision=precision.value,
```

Comment:
Btw, this is just a nit.

Probably safer to pass as string either using the value of str with __str__ implemented since that's the expected type of the argument.

Reply:
True~